bindata/cert-manager-deployment/cert-manager-controller/cert-manager-deployment.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+    labels:
+        app: cert-manager
+        app.kubernetes.io/component: controller
+        app.kubernetes.io/instance: cert-manager
+        app.kubernetes.io/name: cert-manager
+    name: cert-manager
+    namespace: cert-manager
+spec:
+    replicas: 1
+    selector:
+        matchLabels:
+            app.kubernetes.io/component: controller
+            app.kubernetes.io/instance: cert-manager
+            app.kubernetes.io/name: cert-manager
+    template:
+        metadata:
+            annotations:
+                prometheus.io/path: /metrics
+                prometheus.io/port: "9402"
+                prometheus.io/scrape: "true"
```

Comment:
are we using this in monitoring or you need a service monitor?

/cc @s-urbaniak

Reply:
Yes, this specifies the fine-scraping settings. We still need a Service Monitor to enable scraping at all.